src/libraries/System.Private.CoreLib/src/System/Random.Xoshiro256StarStarImpl.cs

```diff
@@ -139,9 +139,7 @@ public override unsafe void NextBytes(Span<byte> buffer)
 
                 while (buffer.Length >= sizeof(ulong))
                 {
-                    Unsafe.WriteUnaligned(
-                        ref MemoryMarshal.GetReference(buffer),
-                        BitOperations.RotateLeft(s1 * 5, 7) * 9);
+                    MemoryMarshal.Write(buffer, BitOperations.RotateLeft(s1 * 5, 7) * 9);
```

Comment:
Does this one get optimized correctly today?

The perf of Random is fairly sensitive and I believe at the time it wasn't well handled.

Reply:
The codegen seesm to be the same - I've checked. We might want to make various helpers like this as "zero budget" for inliner if we hit any issues because of that

Reply:
sounds good; just wanted to confirm that it was getting handled right since its a slightly different change than all the others in the PR